src/data-mgt/node/controllers/transform.js

```diff
@@ -629,18 +628,21 @@ const data = {
                   ...transformedField,
                   errors,
                 };
-
                 let cleanedFinalTransformation = transformUtil.clean(newResp);
 
-                const responseFromConvertFromHectopascalsToKilopascals =
-                  transformUtil.convertFromHectopascalsToKilopascals(
-                    cleanedFinalTransformation.ExternalPressure
-                  );
+                if (cleanedFinalTransformation.ExternalPressure) {
```

Comment:
I also handled an edge case for cases where we carry out metric conversions for External Pressure. Handled the case when a device does not have external pressure.